modules/tools/pages/query-workbench.adoc

```diff
@@ -443,6 +443,50 @@ This simply shows the text output of the {explain}[EXPLAIN] command.
 
 image::query-workbench-result-plantext.png["Query plan in JSON format"]
 
+== Chart
+
+You can visualize query results as a chart, as long as your query returns a suitable data series. Execute the following query in the workbench:
+
+[source, couchbasequery]
+----
+SELECT r.airline, COUNT(1) num_routes,  SUM(ARRAY_COUNT(r.schedule)) schedules
+from `travel-sample`.inventory.route r
+WHERE r.sourceairport = 'SFO'
+GROUP BY r.airline, r.schedule
+----
+
+As well as examining the data in table and JSON formats:
+
+[tabs]
+====
+
+table::
++
+--
+image::query-workbench-chart-table.png[]
+--
+
+json::
++
+--
+image::query-workbench-chart-json.png[]
+--
+====
+
+you can render the results as a chart:
+
+image::query-workbench-chart.png[]
+{empty}
+You can change the type of chart, labels and data set by selecting from the three drop-downs marked above.
+
+[horizontal]
+
+1. You can render the chart in one of several formats: `X-Y`, `Connected Point`, `Line`, `Area`, `Bar`, `Group Bar`, `Pie`, and `Donut`.
+
+2. Select a label for your axis. This must be one of the items returned in your query results.
+
+3. The data series to be rendered on the chart.
```

Comment:
I've used unicode circled numbers as a hack to fake the style of callout lists for images: ➀➁➂. See lines 410-413 above. Not really an issue, It would be good to get the two callout lists to look consistent.

There is an open AsciiDoc issue to be able to create callout lists for images, but it's still in discussion.

Reply:
Yes, that's much better. Thanks. I might change the callouts on the diagram so they look similar.